spec/models/form526_job_status_spec.rb

```diff
@@ -10,7 +10,7 @@
       {
         form526_submission_id: form526_submission.id,
         job_id: jid,
-        job_class: EVSS::DisabilityCompensationForm::SubmitForm526.class.name.demodulize,
```

Comment:
EVSS::DisabilityCompensationForm::SubmitForm526.class.name.demodulize
=> "Class" , which is not correct.

Reply:
If you add .new before .class (EVSS::DisabilityCompensationForm::SubmitForm526.new.class.name.demodulize) does it give you what you want?

Reply:
yeah, that works too.  I just want to test that the value is `SubmitForm526` here.